src/plugins/intel_cpu/src/perf_count.h

```diff
@@ -53,5 +52,10 @@ class PerfHelper {
 }   // namespace intel_cpu
 }   // namespace ov
 
-#define GET_PERF(_node) std::unique_ptr<PerfHelper>(new PerfHelper(_node->PerfCounter()))
-#define PERF(_node, _need) auto pc = _need ? GET_PERF(_node) : nullptr;
+#  define PERF(_node, _need, _itrKey) PERF_COUNTER(_need, PerfHelper, _node->PerfCounter())
+#  define PERF_SHAPE_INFER(_node)
+#  define PERF_PREPARE_PARAMS(_node)
+#endif // ifndef CPU_DEBUG_CAPS
+
+#define GET_PERF(_helper, ...) std::unique_ptr<_helper>(new _helper(__VA_ARGS__))
+#define PERF_COUNTER(_need, _helper, ...) auto pc = _need ? GET_PERF(_helper, __VA_ARGS__) : nullptr;
```

Comment:
Du we still need to allocate the counter on the heap. May be the `nullptr` may be replaced with an "Empty" object, that will reduce the amount of dynamic memory allocations/deallocations, which are not that cheap.